Makefile

```diff
@@ -96,6 +98,10 @@ install-inventory-scripts: install-datadir
 	install -m 755 -d $(prefix)$(datadir)/mender/inventory
 	install -m 755 $(INVENTORY_SCRIPTS) $(prefix)$(datadir)/mender/inventory/
 
+install-inventory-geo-script:
```

Comment:
so, perhaps we could use it and install it by default?

Reply:
The issue is that we'd like to install it conditionally here: https://github.com/mendersoftware/meta-mender/pull/1161. And I don't know how to do this otherwise. Here it is installed by default (as it is now), only, it is possible for more experienced users to leave it out, and in the future, maybe have this as opt in.

Good ideas are always welcome though

Reply:
I think I would split this a bit differently. I agree with @merlin-northern that we should install it by default when running `make`. Here's my proposal:

* `install-inventory-scripts` - Installs all scripts
* `install-local-inventory-scripts` - Installs all scripts except geo
* `install-network-inventory-scripts` - Installs geo
* `install` - Depends on `install-inventory-scripts` and hence installs all inventory scripts.

Reply:
Alright, sounds smrt, I'll do this.

Reply:
:+1: